README.md

```diff
@@ -57,6 +60,8 @@ A list of full blown projects to create a robot vacuum.
 - [I can't belive it's not Valetudo](https://github.com/Hypfer/ICantBelieveItsNotValetudo) - Valetudo companion service (Map Rendering) `Apache-2.0` `JavaScript`
 - [Valetudo RE](https://github.com/rand256/valetudo) - Experimental vacuum software, cloud free (Valetudo Fork) `Apache-2.0` `JavaScript`
 - [valetudo mapper](https://github.com/rand256/valetudo-mapper) - Valetudo companion service (Map Rendering, ICantBelieveItsNotValetudo Fork) `Apache-2.0` `JavaScript`
+- [FreeCong.ga](https://freecon.ga) - Valetudo implementation for Cecotec Conga Vacuums. It has [Home Assistant addon](https://github.com/txitxo0/congatudo-add-on) to make easier how Valetudo is used with your conga.
```

Comment:
A couple of questions:

What is the license of those repositories? Is it the same for all? I have seen a lot of `GPL-3.0` so far. If not they should be separated into multiple entries. Please add the license and main project language as shown in the other examples.

Why does the addon not link to the original repository, but to the fork of @txitxo0 instead?

Whats the exact purpose of each repository? I cannot figure out by the description what OpenDoñita is. How is it different from FreeCong.ga? Do we really need to list all of them?

Reply:
I made some changes to the links.

I used @txitxo0 repository because he is the original author of that addon but yes, it is also in the @freeconga organization so I add it.

All related projects to Freeconga are GPLv3, also the first rastersoft version for Conga 1490.

OpenDoñita was a work of @rastersoft to avoid the usage of chinese servers and Internet dependency of Conga 1490. He has posted how he did his work [in his blog (in Spanish)](https://blog.rastersoft.com/?p=2324).

FreeConga/Valetudo - It is a project to have for more modern models Valetudo (3X90 original work but works as well with 4090, 5090 and 5490 models). Original work was [Badconga](https://github.com/adrigzr/badconga) of @adrigzr. He made the [agnoc](https://github.com/adrigzr/agnoc) library.

All other links in Freeconga website are blog posts or other interesting stuff like this repository.

Please tell me if there is still any other doubt or I can improve more the changes I made.

Thank you @markusressel

Reply:
I have added licenses and languages, will merge for now.
Thx!

Reply:
Thanks to you!